app/recordtransfer/admin.py

```diff
@@ -202,9 +202,29 @@ class UploadSessionAdmin(ReadOnlyAdmin):
     def upload_size(self, obj):
         return get_human_readable_size(obj.upload_size, 1000, 2)
 
-    fields = ["token", linkify("user"), "started_at", "file_count", "upload_size", "status"]
+    fields = [
+        "token",
+        linkify("user"),
+        "started_at",
+        "file_count",
+        "upload_size",
+        "status",
+        "last_upload_interaction_time",
+        "expired",
+        "expires_at",
+    ]
     search_fields = ["token", "user__username"]
-    list_display = ["token", linkify("user"), "started_at", "file_count", "upload_size", "status"]
+    list_display = [
+        "token",
+        linkify("user"),
+        "started_at",
+        "file_count",
+        "upload_size",
+        "status",
+        "last_upload_interaction_time",
+        "expired",
+        "expires_at",
+    ]
```

Comment:
I recommend re-ordering the fields so the change time appears right after the started at time. I also think renaming this field to "Last upload at" for the list display looks a bit cleaner.

```suggestion
    list_display = [
        "token",
        linkify("user"),
        "started_at",
        "last_upload_at",
        "file_count",
        "upload_size",
        "status",
        "expired",
        "expires_at",
    ]
    
    @admin.display(description="Last upload at")
    def last_upload_at(self, obj: UploadSession):
        return obj.last_upload_interaction_time
```

Reply:
I agree with everything.

Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/403/commits/d6e1a7775637863e5d2fae36c914090a98fd7b03